src/lava/utils/profiler.py

```diff
@@ -2,13 +2,40 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # See: https://spdx.org/licenses/
 
-"""
-This module will contain a tool to determine power and performance of workloads
-for Loihi 1 or Loihi 2 based on software simulations or hardware measurements.
-
-The execution time and energy of a workload will be either measured on hardware
-during execution or estimated in simulation. The estimation is based on
-elementary hardware operations which are counted during the simulation. Each
-elementary operation has a defined execution time and energy cost, which is
-used in a performance model to calculate execution time and energy.
-"""
+import warnings
+import typing as ty
+from lava.magma.core.run_configs import RunConfig, Loihi2HwCfg
+
+# Dictionary to relate a certain RunCfg to specific Profiler
+run_cfg_dict: ty.Dict[RunConfig, RunConfig] = {}
```

Comment:
Should this maybe go into the `Profiler` class?

Reply:
It was, but this dict needs to be filled based on available imports. The current solution can probably be improved, but it fits with our current style to try importing loihi specific modules.